src/storage/txn/latch.rs

```diff
@@ -149,9 +194,16 @@ impl Latches {
     where
         H: Hash,
     {
+        // TODO: avoid Hasher construct
```

Comment:
Creating a hasher is very cheap. If we really need to reduce the cost, we must change the hash algorithm